client/src/RatingsAndReviews/Reviews/AddReview.jsx

```diff
@@ -138,20 +139,29 @@ class AddReview extends React.Component {
           <label className="r-title">Write a short review summary</label>
           <input
             className="r-input-small"
-            placeholder="Enter Summary..."
-            stateVal="summary"
+            placeholder="Example: Best purchase ever!"
+            stateval="summary"
             value={this.state.summary}
             onChange={this.handleChange}/>
         </div>
         {/* REVIEW BODY */}
         <div className="r-full">
           <label className="r-title">Write your full review <a className='r-required'>*</a></label>
           <textarea className="r-full-input"
-          placeholder="Enter Review..."
-          stateVal="body"
+          placeholder="Why did you like the product or not?"
+          stateval="body"
           value={this.state.body}
           onChange={this.handleChange}/>
         </div>
+        <div className="r-full-min">
+          {
+            this.state.body
+              ? this.state.body.length + 1 > 49
```

Comment:
Don't think I've seen the double ternary `?` before, if that is what this is. I'll have to check out how it works